composables/data/tagPages.ts

```diff
@@ -0,0 +1,16 @@
+import { TagPage } from './types'
+
+export function normalizeFindTagPageResponse (tagPageResponse: any): TagPage {
+    return tagPageResponse.value.items?.map(normalizeTagPage)
+}
+
+export function normalizeTagPage(page: Record<string, any>): TagPage {
```

Comment:
Could also be simplified

Reply:
As above, I'm intentionally redefining the data here. In particular in this case I want to fix the capitalization discrepancy in `topPageZone` and `midpageZone` which really bugs me.(`top_page_zone` and `midpage_zone` on the api response)